nulogy/design-system
# Adds guides for publishing packages to npm as well improving component docs

The PR updates some documentation:

1. Improves the [`components/README.md`](https://github.com/nulogy/design-system/blob/f4104147ce18c2fee17ef91d95ade1f95e5e1a7a/components/README.md)
2. Adds that readme as the [intro to the Components section](https://deploy-preview-40--nulogy-design-system.netlify.com/components) of the docs.
1. Updates the [Welcome](https://deploy-preview-40--nulogy-design-system.netlify.com) page (and project [README](https://github.com/nulogy/design-system/blob/f4104147ce18c2fee17ef91d95ade1f95e5e1a7a/README.md)) to link to the Components section of the docs.
3. Adds a [Guides/Packages](https://deploy-preview-40--nulogy-design-system.netlify.com/guides/packages) section to the docs explains what all the packages are.
4. Adds a [Guides/Publishing packages to npm](https://deploy-preview-40--nulogy-design-system.netlify.com/guides/publishing) section to the docs.

# To test

Take a look at the [deploy preview](https://deploy-preview-40--nulogy-design-system.netlify.com). 

Note that links are all absolute links to the http://nulogy.design website and will be broken in the deploy preview.
